src/app/code/community/Shopgate/Framework/etc/config.xml

```diff
@@ -216,16 +216,6 @@
                 </observers>
             </sales_order_place_after>
 
-            <sales_quote_address_collect_totals_before>
```

Comment:
I did not get much into the framework solution, this stuff still works without this and without cloud plugin?